artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/broker/ProtonProtocolManager.java

```diff
@@ -78,6 +80,8 @@
 
    private boolean amqpUseModifiedForTransientDeliveryErrors = AmqpSupport.AMQP_USE_MODIFIED_FOR_TRANSIENT_DELIVERY_ERRORS;
 
+   private String mirrorAddress = "$mirror";
```

Comment:
I will call this $artemis.mirror and have a constant.

not customizing it would mean it's a lot easier to document it which I'm actually struggling to explain this concept.

Reply:
That seems reasonable. It can always be made configurable at a later point if someone estbalished a particular need for it.

(I totally forgot to think back on these points as I had commented, as it took waaaaay longer than expected to reivew the rest :P)